src/package/packageVersionReport.ts

```diff
@@ -32,6 +32,51 @@ const QUERY_VERBOSE =
   'ORDER BY Package2Id, Branch, MajorVersion, MinorVersion, PatchVersion, BuildNumber';
 
 const logger = Logger.childFromRoot('getPackageVersionReport');
+
+/**
+ * Given a list of subscriber package version IDs (04t), return the associated version strings (e.g., Major.Minor.Patch.Build)
+ *
+ * @return Map of subscriberPackageVersionId to versionString
+ * @param subscriberPackageVersionIds
+ * @param connection For tooling query
+ */
+// eslint-disable-next-line @typescript-eslint/require-await
+async function getPackageVersionStrings(
```

Comment:
why keep this async, if it's not required to be?

Reply:
I believe I need to remove the eslint directive. Line 63 is using await.